decentraland/unity-explorer
# Fix: duplicate available resolutions

## What does this PR change?


Fix #2264 

Prevent the addition of duplicate resolutions in the menu settings


## How to test the changes?


This issue was reproducible with a multiple monitor setup where at least 2 monitors share the specs (resolution and refresh rate)

1. Launch the explorer from a different build (to make sure that your testing setup has the issue)
2. Go to settings
3. Check that there are duplicate resolutions
4. Launch the explorer from this build
5. Go to settings
6. Check that there aren't any duplicate resolutions

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
